animation_nodes/nodes/sound/sound_intensity.py

```diff
@@ -0,0 +1,61 @@
+import bpy
+import numpy
+from bpy.props import *
+from ... data_structures import DoubleList
+from ... base_types import AnimationNode, VectorizedSocket
+
+reductionItems = [
+    ("MEAN", "Mean", "", "", 0),
+    ("MAX", "Max", "", "", 1)
+]
+reductionFunctions = {
+    "MEAN" : numpy.mean,
+    "MAX" : numpy.max
+}
+
+class SoundIntensityNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_SoundIntensityNode"
+    bl_label = "Sound Intensity"
+    errorHandlingType = "EXCEPTION"
+
+    smoothingSamples: IntProperty(name = "Smoothing Samples", default = 5, min = 0)
+    reductionFunction: EnumProperty(name = "Reduction Function", default = "MAX",
+        items = reductionItems)
+
+    useFloatList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput("Sound", "Sound", "sound")
+        self.newInput(VectorizedSocket("Float", "useFloatList",
+            ("Frame", "frame"), ("Frames", "frame")))
+        self.newInput("Float", "Attack", "attack", value = 0.005, minValue = 0, maxValue = 1)
+        self.newInput("Float", "Release", "release", value = 0.6, minValue = 0, maxValue = 1)
+        self.newInput("Scene", "Scene", "scene", hide = True)
+
+        self.newOutput(VectorizedSocket("Float", "useFloatList",
+            ("Intensity", "intensity"), ("Intensities", "intensities")))
+
+    def drawAdvanced(self, layout):
+        layout.prop(self, "reductionFunction", text = "")
+        layout.prop(self, "smoothingSamples")
+
+    def getExecutionFunctionName(self):
+        if self.useFloatList: return "executeMultiple"
+        else: return "executeSingle"
+
+    def executeSingle(self, sound, frame, attack, release, scene):
+        if len(sound.soundSequences) == 0: self.raiseErrorMessage("Empty sound!")
+
+        fps = scene.render.fps
+        return sound.computeTimeSmoothedIntensity(frame / fps, (frame + 1) / fps,
```

Comment:
I'm still unsure if `frame / fps, (frame + 1) / fps` is really the right approach to calculate the windows, but I guess we can leave it as is for now..

Reply:
It might make sense to ensure a certain spectral resolution in the Sound Spectrum node as you suggested. However, I don't see why that would make sense for the sound intensity node. The intensity is computed directly from the samples and have no consequence on low sample counts, in fact, it is more accurate.

I tried with multiple frame rates and the result was nice, so I agree, we should leave this for now.